schematic/store/synapse.py

```diff
@@ -1537,14 +1540,21 @@ async def format_row_annotations(
                 annos.pop(anno_k) if anno_k in annos.keys() else annos
             # Otherwise save annotation as approrpriate
             else:
+                if annotation_keys == "display_label":
```

Comment:
I think you'll find this video very interesting @GiaJordan https://www.youtube.com/watch?v=CFRhGnuXG-4

They take it to an extreme, however, the principals are solid. I find that there is a good in-between where you are extracting out nested logic into it's own functional blocks, but not being so strict that simple things need to have the extra boilerplate code.

Clarity of the code helps debugging quite a lot. It can also lead to identifying better/different solutions to the problem.